packages/backend-for-frontend/src/services/clientService.ts

```diff
@@ -249,6 +254,14 @@ export function clientServiceBuilder(
           )
         )
       );
+
+      return name
+        ? users.filter((user) =>
+            new RegExp(ReadModelRepository.escapeRegExp(name), "i").test(
```

Comment:
This function is duplicated, [regexpUtils.ts](https://github.com/pagopa/interop-be-monorepo/blob/045ecd63455f6f13714e1d5c557a4cd55f0a7247/packages/commons/src/utils/regexpUtils.ts#L1) and [ReadModelRepository.ts](https://github.com/pagopa/interop-be-monorepo/blob/045ecd63455f6f13714e1d5c557a4cd55f0a7247/packages/commons/src/repositories/ReadModelRepository.ts#L233) (it must be unified in regexpUtils).

In this case, you’re using this function as a generic utility, so I suggest using the version from regexpUtils.ts. This way, you can avoid importing ReadModelRepository, which is outside the context of the clientService.ts file.